conf/locale/babel_mako.cfg

```diff
@@ -1,13 +1,30 @@
-# Extraction from Mako templates
-[mako: cms/templates/**.html]
+# Extraction from Mako templates.
+# Mako files can appear in a number of different places. Some we want to
+# extract strings from, some we don't.
+#
+# Extract from these directory trees:
+#
+#   cms/templates
+#   cms/djangoapps/APPNAME/templates
+#   common/templates
+#   common/djangoapps/APPNAME/templates
+#   lms/templates
+#   lms/djangoapps/APPNAME/templates
+#   common/lib/capa/capa/templates
+#
+# Don't extract from these directory trees:
+#   common/test/test_microsites/test_microsite/templates
+#
+
+[mako: */templates/**.html]
 input_encoding = utf-8
-[mako: lms/templates/**.html]
+[mako: */djangoapps/*/templates/**.html]
```

Comment:
It looks like this won't match our teams Mako file:

    lms/djangoapps/teams/templates/teams/team.html

I think maybe you need an extra level in the rule:

    */djangoapps/*/templates/*/**.html

Note that we have a namespace inside the directory so that Django doesn't get URI collisions. See this page for details:

https://openedx.atlassian.net/wiki/display/AC/How+to+add+a+new+feature+to+LMS+or+Studio